Week3/Week3-JS-I.js

```diff
@@ -27,145 +27,146 @@ const newModulo = 21 % 5 === null;
 
 // 7
 function returnString(str) {
-  //simply return the string provided: str
+  return str;
 }
 
 // 8
 function add(x, y) {
-  // x and y are numbers
-  // add x and y together and return the value
-  // code here
+  return x + y;
 }
 
 // 9
 function subtract(x, y) {
-  // subtract y from x and return the value
-  // code here
+  return x - y;
 }
 
 // 10
 function multiply(x, y) {
-  // multiply x by y and return the value
-  // code here
+  return x * y;
 }
 
 // 11
 function divide(x, y) {
-  // divide x by y and return the value
-  // code here
+  return x % y;
 }
 
 // 12
 function areEqual(x, y) {
-  // return true if x and y are the same
-  // otherwise return false
-  // code here
+  if (x === y) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 13
 function areSameLength(str1, str2) {
-  // return true if the two strings have the same length
-  // otherwise return false
-  // code here
+  if (str1.length === str2.length) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 14
 function lessThanNinety(num) {
-  // return true if the function argument: num , is less than ninety
-  // otherwise return false
-  // code here
+  if (num < 90) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 15
 function greaterThanFifty(num) {
-  // return true if num is greater than fifty
-  // otherwise return false
-  // code here
+  if (num > 50) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 16
 function getRemainder(x, y) {
-  // return the remainder from dividing x by y
-  // code here
+  return x % y;
 }
 
 // 17
 function isEven(num) {
-  // return true if num is even
-  // otherwise return false
-  // code here
+  if (num % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
 }
 
 // 18
 function isOdd(num) {
-  // return true if num is odd
-  // otherwise return false
-  // code here
+  if (num % 2 === 1) {
+    return true;
+  } else {
+    return false;
+  }
 }
-
-// 19
+//19
 function square(num) {
-  // square num and return the new value
-  // hint: NOT square root!
-  // code here
+  return num * num;
 }
-
 // 20
 function cube(num) {
-  // cube num and return the new value
-  // code here
+  return Math.pow(num, 3);
 }
 
 // 21
 function raiseToPower(num, exponent) {
-  // raise num to whatever power is passed in as exponent
-  // code here
+  return Math.pow(num, exponent);
 }
 
 // 22
 function roundNumber(num) {
-  // round num and return it
-  // code here
+  return Math.ceil(num);
```

Comment:
There is another method that you should use here to round the number in either direction. We don't want the number to round up by defaults. Try this one again.